data-access/src/app/application-services/community/community.domain.ts

```diff
@@ -43,17 +43,28 @@ export class CommunityDomainApiImpl
     if (this.context.verifiedUser.openIdConfigKey !== 'AccountPortal') {
       throw new Error('Unauthorized');
     }
-
+    const vendorsList = []
+    if(community.approvedVendors){
+      const {approvedVendors} = community;
+      for(const approvedVendor of approvedVendors){
+        const vendor = await this._context.applicationServices.users.vendorUser.dataApi.getUserById(approvedVendor.id);
+        const vendorDo = new VendorUserConverter().toDomain(vendor, ReadOnlyInfrastructureContext(), ReadOnlyDomainExecutionContext());
+        vendorsList.push(vendorDo);
+      }
+    }
+    const updatedCommunity = {...community, approvedVendors: vendorsList}
```

Comment:
This isn't necessary, remove updatedCommunity and use community below. For ApprovedVendors setter, just use vendorsList as the argument